dask/bytes/core.py

```diff
@@ -90,6 +91,9 @@ def read_bytes(
         represented in the corresponding block.
 
     """
+    if not isinstance(urlpath, (str, list, tuple, Path)):
```

Comment:
Changing `Path` to `os.PathLike` should allow anything that implements `__fspath__`.

I don't know when `os.PathLike` was added to the stdlib. If it's not present for py36 then you can use `hasattr(urlpath, '__fspath__')`.

Reply:
TIL
It is in py36 so I'll do that.